mdn/kuma
# Add new newsletter signup to register and user edit

Fixes #6995, fixes #6996 

Since signup was basically just a copy change, and user edit looks the same, I decided to do them in bulk (and also do the sendinblue stuff in a separate PR).

## How to test it
Create a new account, do or don't signup to the newsletter. Check edit profile to see that your setting has been retained and that you can update it.

![Screenshot_2020-05-11 Create your MDN Web Docs Account](https://user-images.githubusercontent.com/4051932/81546705-0efddc80-937b-11ea-9bfe-a0eef9630b40.png)
![Screenshot_2020-05-11 Edit Your Profile MDN](https://user-images.githubusercontent.com/4051932/81546711-11603680-937b-11ea-9d9f-0dd9f6e3a3aa.png)
